app/helpers/media_helper.rb

```diff
@@ -3,11 +3,43 @@ def media_thumbnail(media_item)
     if media_item.thumbnail?
       image_tag(media_item.item.thumb.url)
     else
-      media_extension = File.extname(media_item.item.file.path).downcase
-      # content_tag(:div, class: "generic-thumbnail") do
-        content_tag(:div, media_extension, class: "extension")
-        # concat(image_tag("file-blank.png"))
-      # end
+      content_tag(:div, class: "media-block") do
+        concat(content_tag(:div, media_item.kind_value.capitalize))
+        concat(sanitize("<i class='fa fa-#{media_icon_class(media_item)}'></i>", tags: %w(i),
+          attributes: %w(class)))
```

Comment:
Or even better would be to add a general helper to application_helper.rb called `icon_tag` that takes the part after `fa-` as an argument and gives back the icon tag. You could use the above within there...